deploy/operator.yaml

```diff
@@ -12,11 +12,11 @@ spec:
       labels:
         name: meta-operator
     spec:
-      serviceAccount: meta-operator
+      serviceAccountName: meta-operator
```

Comment:
Do we need to change code after the spec change? @DanielXLee @horis233

Reply:
<img width="1130" alt="Screen Shot 2020-02-09 at 9 21 56 PM" src="https://user-images.githubusercontent.com/33276414/74116586-3e395d80-4b82-11ea-84fb-481fcba34173.png">
`serviceAccount` is depreciated, so I change it to `serviceAccountName`.
I don't think we need to change the operator code for it.